### ISSUE TITLE
OS Home Assistant does not start until the Eufy Security addon starts

### ISSUE THREAD

**Issue by DivanX10** [fuatakgun/eufy_security#157]
**Describe the bug**
The problem has not been solved and for some reason you have [closed this branch](https://github.com/fuatakgun/eufy_security/issues/75). Reopened the topic. Please do not close the topic until the problem is fixed. The Eufy Security addon does not allow Home Assistant to complete the full download until the Eufy Security addon is launched. The problem is serious and for me it is even critical.  Why is the problem critical? If the lights went out in the house, then Synology turns off virtual machines when switching to safe mode, and after powering up Synology starts virtual machines and Home Assistant cannot start until I manually launch the Eufy Security addon myself. 

**By observation, I found the reason that causes the problem that the addon cannot start**
> When I install this Eufy Security integration, then after completely turning off the Home Assistant and turning it on, the addon does not start, and if I remove the Eufy Security integration and turn off the Home Assistant completely, and then turn it on, the Eufy Security addon starts normally.

**To Reproduce**
It behaves similarly on the VMware Workstation virtual machine. You can repeat this on VMware Workstation
1. Turn off OS Home Assistant completely via host shutdown
2. Turn on OS Home Assistant
3. During the download, Home Assistant will wait for the Ugan Security addon to start

**Expected behavior**
When the Home Assistant is fully loaded, the Eufy Security addon should start immediately, and the Home Assistant should start working

**Screenshots**
![image](https://user-images.githubusercontent.com/64090632/144294823-f8c22aea-8c19-43b8-b47c-5b522bc71a4f.png)
![image](https://user-images.githubusercontent.com/64090632/145671449-097efc55-a867-4ecf-a49c-2820b1b575fc.png)

**Hardware Information about Home Assistant**
NAS Synology

**Comment by fuatakgun** [comment]
I asked in previous issue to share your updates, no need a separate issue, i can reopen them based on your comments.

**Comment by DivanX10** [comment]
Yes, it has worked fine since the latest version. Now the addon does not prevent the Home Assistant from loading quickly. Thanks.
